nixos/modules/services/networking/minidlna.nix

```diff
@@ -36,6 +36,36 @@ in
         '';
     };
 
+    services.minidlna.friendlyName = mkOption {
+      type = types.str;
+      default = "${config.networking.hostName} MiniDLNA";
+      example = "rpi3";
+      description =
+        ''
+          Name that the DLNA server presents to clients.
+        '';
+    };
+
+    services.minidlna.rootContainer = mkOption {
+      type = types.str;
+      default = ".";
+      example = "B";
+      description =
+        ''
+            Use a different container as the root of the directory tree presented
+            to clients. The possible values are:
+            ,* "." - standard container
```

Comment:
Will this be formatted OK?

Reply:
No idea, i pasted as is from minidlna's documentation.
Will simplify this.